docs/lab-3/README.md

```diff
@@ -4,17 +4,13 @@ description: Learn about prompt engineering techniques
 logo: images/ibm-blue-background.png
 ---
 
-!!! note
-    This lab was informed by [@juliandiscovers](https://www.youtube.com/channel/UCXKuHxwAeZZu_dg_qs8Z-9w) and [@BenzaMaman](https://www.instagram.com/benzamaman/)'s workshops and content and presents a broad overview of what they teach. If you want to dive deeper into Prompt Engineering we recommend checking them out.
```

Comment:
I think you should keep this in here, i really did take a lot of their content for this lab.